README.md

```diff
@@ -7,12 +7,16 @@ Reference:
 
 ## Quick Start
 
-1. `npx create-next-app@latest my-app`
+1. `npx create-next-app@13.1.5 my-app`
```

Comment:
Could you put `13.0.0` here and below?

I did try using `13.1.5` but it seems like Next using app directory doesn't work because they started relying on the AsyncLocalStorage (which we don't yet support: https://github.com/cloudflare/next-on-pages/issues/76), so I think for now it's safer to just do `13.0.0`